src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt

```diff
@@ -100,7 +101,14 @@ class JoarkMottak(val env: Environment, private val journalpostArkiv: Journalpos
 
     private fun hentInngåendeJournalpost(journalpostId: String): Behov {
         val journalpost = journalpostArkiv.hentInngåendeJournalpost(journalpostId)
-        jpCounter.inc()
+        jpCounter
+            .labels(
+                journalpost.tema,
```

Comment:
minor - but `tema` will always be `DAG` as we filter out other "temas" in the kafka stream.